float.css

```diff
@@ -1,6 +1,7 @@
 *{
```

Comment:
why remove the *{} in float? that's how thinkful's example was. can you
tell me why?

On Mon, Aug 22, 2016 at 7:08 AM, khanh nguyen <notifications@github.com>
wrote:

> In float.css
> <https://github.com/Jenn083/Jane-Doe/pull/1#discussion_r75683691>:
>
> > @@ -1,6 +1,7 @@
> >  *{
>
> It is not good to use universal selector. Please remove this selector and
> have specific element or class you would like to apply to.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Jenn083/Jane-Doe/pull/1/files/84e5654ea8afd470360afaa0c9c4464ee1ea1179#r75683691>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHHcIOTpsG32N-bECKTZiY2rRrus2Mufks5qia1LgaJpZM4Jp2_6>
> .
>